.gitignore

```diff
@@ -138,6 +138,7 @@ static/
 
 # doc
 doc/source/savefig/
+*.txt
```

Comment:
Is this change relavent to the demo?

Reply:
I put the query document in the repo initially, but I think you are right we don't need to make this change. I will delete it.